ci/build.py

```diff
@@ -33,6 +33,7 @@
 import shutil
 import subprocess
 import sys
+import docker_cache
```

Comment:
Some users are asking if mxnet maintains any official docker containers. This is probably only used for CI, but i guess some of these procedures can be reused if we want to host official docker containers?